### DISCUSSION TITLE
Passport::routes() no longer defined - but where did the passport routes go?

### DISCUSSION THREAD

**Discussion by Synchro** [laravel/framework#46295] [Q&A] [answered]
I've run into this while upgrading to L10 - the `Passport::routes()` method no longer exists, and [the upgrade guide recommends removing it](https://github.com/laravel/passport/blob/master/UPGRADE.md#refactor-routes-to-dedicated-file). However, it also says that it moves the routes to the `web.php` routes file, but I can find no sign of this happening. [The PR that made this change](https://github.com/laravel/passport/pull/1464) says the same thing. I just tried creating new L9 and 10 apps with passport, but there is no sign of passport routes in any of the routes files – `web.php` just contains the default welcome route. Are these routes created internally so that they do not need to appear in routes files unless they are being overridden? If they should be appearing, is their absence a bug?

**Comment by ahmett** [comment, marked as answer]
The routes you are looking for are defined and registered at the package level.

_Please see:_
[routes/web.php](https://github.com/laravel/passport/blob/11.x/routes/web.php)
[src/PassportServiceProvider.php](https://github.com/laravel/passport/blob/11.x/src/PassportServiceProvider.php#L39)

**Reply by Synchro** [reply]
Ah, ok, so there is a *different* `web.php` file, in addition to the one in my app's `routes/web.php` file. That wasn't clear from the docs.
